example.jsonnet

```diff
@@ -30,11 +30,27 @@ local s = t.store(commonConfig.config {
   serviceMonitor: true,
 });
 
+local split = t.receiveSplit(commonConfig.config {
```

Comment:
I think we should rather have two additional helper functions: `receiveRouter` and `receiveIngestor` in addition to the existing `receive`. This would be more clear and allow just o configure those layers independently. After having those maybe we can have a convenience function like `receiveSplit`

Reply:
Aha that is a nice separation of concerns. The only trickiness then is the dependencies between them - router **needs** to know about ingestor's stores - so it kind of does not make sense to allow them to be consumed separately? WDYT?

Reply:
I think rather than burying this connection under a `receiveSplit` function we can make it visible. This is already an advanced feature, the users should know what they are doing at this stage TBH.

Moreover, I would rather have the ability and flexibility to configure and mix/mash these. In the end, `kube-thanos` is a library for people to extend in their downstream projects.

Reply:
Split them out - `example.jsonnet` includes demonstration about how they are combined :+1: